### PR TITLE
Installation instruction added

### PR DESCRIPTION
**Pull request by madhusudhan1234**

### CODE REVIEW

File: Readme.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+### Installation Instruction
+- Clone The repository using command ```$ git clone https://github.com/sadhakbj/Laravel-vue-spa-vuex.git```
+- Create database on mysql
+- Create .env file and add database detail on it
+- Go to the repository using cd command and run command ```$ composer install```
+- Run migrate command ```$ php artisan migrate```
+- Run ```$ php artisan server```, Your application is running on port 8000 
+- You can get you application ```http://localhost:8000``` in browser. Cheers!
+
+### No Man I want to use docker
```
Comment (sadhakbj) [inline Readme.md]: What about the javascript ?

Comment (madhusudhan1234) [reply Readme.md]: Haha, I forgot.

Comment (madhusudhan1234) [reply Readme.md]: @sadhakbj I made changes now

**Comment by sadhakbj** [comment]
Reopened.
.........

**Pull request closed without merging**